scheduler-worker/src/test/java/gov/nasa/jpl/aerie/scheduler/worker/services/SchedulingIntegrationTests.java

```diff
@@ -3154,7 +3102,7 @@ void testOptionalSimulationAfterGoal_staleResources() {
           List.of(
               new SchedulingGoal(new GoalId(0L, 0L), """
                   export default () => Goal.CoexistenceGoal({
-                    forEach: ActivityExpression.ofType(ActivityTypes.GrowBanana),
+                    forEach: Real.Resource("/fruit").greaterThan(4),
```

Comment:
because of the mocking of simulation results in case of activity dependency only in `PrioritySolver.getLatestSimResultsUpTo` which make this goal do something even without simulation.